src/js/tech/tech.js

```diff
@@ -238,6 +238,14 @@ class Tech extends Component {
    * @method dispose
    */
   dispose() {
+    // clear out text tracks because we can't reuse them between techs
```

Comment:
So the text track list (player.textTracks) will survive. We'll remove the old text tracks (which will trigger change events) and add new text tracks (which will also trigger change events). Users should listen for those events to handle re-adding listeners to the new tracks. Is that right?

That doesn't sound too bad.